examples/Mass Update User Zones/README.md

```diff
@@ -0,0 +1,87 @@
+# Mass Update User Zones #
+
+## Overview ##
+The Mass Update User Zones script utilizes the Box CLI to provision users to the specified data residency Zone within a Multizone Box tenant.
+
+The script takes a .csv containing users and zone mappings as input and makes the zone assignment as specified. It also uses the admin or co-admin login email address to look up the associated enterprise and the zone policy assigned to the enterprise (which is inherited by all users unless otherwise specified; sometimes referred to as the "default zone"). 
+
+Zone assignments are interpreted using the `ZonesTable`.  This script is designed to be run once to do the initial provisioning of user zones, however, it can also be used for subsequent runs to make zone updates ad hoc.
+
+If you are wondering how to assign Zones through the Admin Console, see [here](https://support.box.com/hc/en-us/articles/360044193533-Assigning-Zones-through-the-Admin-Console) for more details. For more information about Box Zones, please visit our [page here](https://www.box.com/zones).
+
+## Setup Pre-Requisites
+1. Clone this github repo.
+2. Install PowerShell or .Net core.
+   > If you encounter issues make sure you install both dotnet core and PowerShell
+    1. For MacOS & Linux, Install the latest version of [PowerShell](https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell?view=powershell-7.2).
+    2. For Windows, Install the latest version of [dotnet core](https://dotnet.microsoft.com/download).
+    
+3. Test PowerShell by running the `pwsh` command in your terminal.
+
+```
+mikechen@mbp create-users-automation % pwsh
+PowerShell 7.2.1
+Copyright (c) Microsoft Corporation.
+
+https://aka.ms/powershell
+Type 'help' to get help.
+
+PS /Users/mikechen>
+```
+
+4. Setting up the CLI requires creating a Box application that uses server authentication with JWT. See the [Box CLI Using JWT Authentication](https://developer.box.com/guides/tooling/cli/jwt-cli/) page for instructions. 
+    * **The Box application must have the following application access level enabled** in the configuration page: `App + Enterprise Access`
```

Comment:
for readability suggest to bold only access level, application scopes, advanced features

Reply:
Done